front/src/app/auth/register/register.component.ts

```diff
@@ -116,6 +128,8 @@ export class RegisterComponent implements OnInit, AfterViewInit {
   }
 
   ngOnInit(): void {
+    this.startDate = new Date();
+    this.startDate.setFullYear( this.startDate.getFullYear() - 35 );
```

Comment:
Why that?

Reply:
When you initialize DatePicker, it will open by default at the current year. Chances are that the majority of our customers aren't born this year :) So I just open it 35 years ago, when you use the DatePicker, you get this:
![image](https://user-images.githubusercontent.com/3328875/107768883-68929d80-6d37-11eb-9f13-8699424efb36.png)